client/my-sites/plugins/plugin-details-sidebar/index.jsx

```diff
@@ -64,6 +66,14 @@ const PluginDetailsSidebar = ( {
 
 	const isPremiumVersionAvailable = !! premium_slug;
 	const premiumVersionLink = `/plugins/${ premium_slug }/${ selectedSite?.slug || '' }`;
+	const premiumVersionLinkOnClik = useCallback( () => {
+		recordTracksEvent( 'calypso_plugin_details_premium_plugin_link_click', {
+			current_plugin_slug: slug,
+			premium_plugin_slug: premium_slug,
+			premium_plugin_link: premiumVersionLink,
```

Comment:
What's the rationale of having the `premiumVersionLink` in the event properties?

Reply:
I don't have a rationale 😅, I basically added it because I have it. Do you suggest to remove?